rebasebot/bot.py

```diff
@@ -174,7 +174,7 @@ def _do_rebase(gitwd, source: GitHubBranch, dest: GitHubBranch, source_repo, tag
     for commit in commits.splitlines():
         # Commit contains the message for logging purposes,
         # trim on the first space to get just the commit sha
-        sha, commit_message, committer_email = commit.split(" - ")
+        sha, commit_message, committer_email = commit.split(" || ")
```

Comment:
any chance of a splitn?

Reply:
could just change to

```suggestion
        sha, commit_message, committer_email = commit.split(" || ", 1)
```

that will give a maximum of 1 split, array size of 2 is returned if successful

Reply:
Added `split(x , 2)`